index.html

```diff
@@ -9,26 +9,24 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<div class="container">
     <div class="main">
         <div class="photo">
-            <img src="images/photo.svg">
+            <img src="images/photo.svg" alt="Face photo">
         </div>
-        <div class="inside-div">
-            <div class="child-div-1">
-                <p class="name">Aleksander</p>
-                <p class="surname">KOWALSKI</p>
-                <p class="position">STANOWISKO</p>
+        <div class="personal-details">
+            <div class="name-and-surname-and-position">
+                <div><p class="name">Aleksander</p></div>
+                <div><p class="surname">KOWALSKI</p></div>
+                <div><p id="position">STANOWISKO</p></div>
             </div>
-            <div class="child-div-2">
+            <div class="contact-and-address">
```

Comment:
The indentation of the closing `</div>` tag of the `contact-and-address` div is wrong